docs/io/visualization/convergence_plot.ipynb

```diff
@@ -0,0 +1,846 @@
+{
```

Comment:
<p>Is changing the layout here necessary? I think this cell should only focus on show them up since you're already describing how to change layout below</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1636/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>